pkg/kdump/kdump.scss

```diff
@@ -31,3 +32,7 @@
   max-block-size: 200px;
   overflow-y: auto;
 }
+
+.service-link-ct-kdump {
+  margin-inline-start: var(--pf-v5-global--spacer--sm);
+}
```

Comment:
We could use the utility class `pf-v5-u-ml-sm` instead, now that PatternFly supports RTL.

That helper class adds this, according to Firefox's devtools:

```css
.pf-v5-u-ml-sm {
  -webkit-margin-start: var(--pf-v5-global--spacer--sm) !important;
  margin-inline-start: var(--pf-v5-global--spacer--sm) !important;
}
```

(`-webkit-margin-start` is an unneeded alias that they're using for whatever reason. The intent is probably to support even older browsers... but Safari added `margin-inline-start` in 2019, Chrome in 2018, Edge in 2020 (when Edge switched to Chromium), and Firefox way back in 2015. So it's really very well supported and not needed. At least by us, and probably really not by PatternFly either.)

Reply:
That works well indeed! It gets broken (no spacing at all) with dropping the custom class, and looks fine again with the pf helper class. Done.